proto/keyspace.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package keyspace;
+
+enum KeyspaceMode {
+    // RawKv mode.
+    Raw = 0;
+    // RawKv mode with TTL.
+    RawTTL = 1;
+    // Transacation mode.
+    Transaction = 2;
+}
+
+message KeyspacePrefix {
```

Comment:
Why make it a separate message?

Reply:
Because keyspace can have **many** prefixes and can bind/unbind prefix.